packages/core/src/request-builder/header-builder/headers-util.ts

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { MapType } from '@sap-cloud-sdk/util';
+
+export function isNullish(x: any): x is null | undefined {
+  return x === null || x === undefined;
+}
+
+export function toHeaderObject(key: string, value: any): MapType<any> {
+  return isNullish(value) ? {} : { [key]: value };
+}
+
+export function getHeader(
+  key: string,
+  headers: MapType<any> = {}
+): MapType<any> {
+  const entry = Object.entries(headers).find(
+    ([entryKey]) => entryKey.toLowerCase() === key.toLowerCase()
+  );
+  return entry ? { [entry[0]]: entry[1] } : {};
+}
+
+export function getHeaderValue(
+  key: string,
```

Comment:
could you use the getHeader() method inside here?

Reply:
sure